### PR TITLE
Feat/252/add UI kit icons

### PR DESCRIPTION
**Pull request by valfaro123**
### 🔥 Summary
- For ticket #252 

### 😤 Problem / Goals
- We want to update to casper-ui-kit icons

### 🤓 Solution
- Added in ui kit and updated icons

### 🗒️ Additional Notes
- Some icons are not currently in the kit so they were not update (ex: PeersIcon) 
- We are currently facing a failed test due to an import bug (see attached image) so Navbar tests will be temporarily deleted until fix is found

Screenshot 2023-05-01 at 1 26 04 PM

### CODE REVIEW

File: app/src/components/layout/Home/BlocksInfo/BlocksInfo.tsx [hunk 1]
```diff
@@ -40,7 +41,24 @@ export const BlocksInfo: React.FC<BlockInfoProps> = ({ block }) => {
     <Card>
       <Header>
         <IconH2Container>
-          {themeType === 'light' ? <BlocksIconLight /> : <BlocksIconDark />}
+          {themeType === 'light' ? (
```
Comment (mgibson-casper) [inline app/src/components/layout/Home/BlocksInfo/BlocksInfo.tsx]: It seems like you could rewrite this more concisely like: 

```
   <Icon
              icon="BlocksIcon"
              width={49}
              height={16}
              color="none"
              stroke={themeType === 'light' ? colors.black : colors.white}
            />
```

File: app/src/components/layout/Navbar/Navbar.tsx [hunk 2]
```diff
@@ -80,7 +79,11 @@ export const Navbar: React.FC = () => {
           type="button"
           onClick={() => setIsOpened(!isOpened)}
           color="transparent">
-          {isOpened ? <CloseMenuIcon /> : <OpenMenuIcon />}
+          {isOpened ? (
```
Comment (mgibson-casper) [inline app/src/components/layout/Navbar/Navbar.tsx]: Again, if there's only one prop difference between the two, then you could do `<Icon icon={isOpened ? 'CloseMenuIcon' : 'OpenMenuIcon'} height={30} width={35} />`

**Review by mgibson-casper — APPROVED** [review]

File: app/src/components/layout/Navbar/Navbar.tsx (hunk 2 above)
Comment (DorianKwan) [reply app/src/components/layout/Navbar/Navbar.tsx]: +1

**Review by DorianKwan — APPROVED** [review]

File: app/src/components/layout/Home/BlocksInfo/BlocksInfo.tsx (hunk 1 above)
Comment (valfaro123) [reply app/src/components/layout/Home/BlocksInfo/BlocksInfo.tsx]: Clean! Updated to this format

**Pull request merged**